core/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java

```diff
@@ -784,6 +785,13 @@ public SmoothingModel readPhraseSuggestionSmoothingModel() throws IOException {
     }
 
     /**
+     * Reads a {@link ValueFormat} from the current stream.
+     */
+    public ValueFormat readValueFormat() throws IOException {
```

Comment:
Should we add a test of this method in BytesStreamTests? (see comment here: https://github.com/elastic/elasticsearch/pull/17351#discussion_r57492046)